### ISSUE TITLE
Cannot create unknown type '{clr-namespace:ActivityLib;assembly=CoreActivities}CodeActivity1'.' 

### ISSUE THREAD

**Issue by lakani** [UiPath/CoreWF#142]
currently to workaround the lake of the designer,
i've  developed 4 projects

Proj1       -> a non dotnet core lib project which contains my Activity class, just a mocked class to be able to use in WF designer 
Proj2       -> a non dotnet core console project which contains my XAML file which reference activity in Proj1 
Proj3       -> a dotnet core lib project which contains a reference to Activity class in (Proj1) 
Proj4       -> a dotnet core console project which contains a reference to Activity class in (Proj3)

Both (Proj1 and Proj2) as non core dotnet wf solution works fine, but when i tries to run Proj4 , i;m getting the following exception 

[ProcessActivity_Error.txt](https://github.com/UiPath/CoreWF/files/6141512/ProcessActivity_Error.txt)

which as i guess, that error raised while loading the XAML which contains Activity cant be loaded by CoreWF although both  Proj1 and Proj3 is identical in namespace and assembly name, 

in other words, how can i make use of Activity lib which is based on CoreWF in WF designer hosted by WWF

you can reference the complete code in my repo   ( https://github.com/lakani/TestCoreWF )

**Comment by lbargaoanu** [comment]
https://github.com/UiPath/CoreWF/discussions/136
